kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -67,7 +67,8 @@ function showExploreContent(store, id) {
 
   Resources.getModel(id).fetch()
     .then((attributes) => {
-      const pageState = _contentState(attributes);
+      const pageState = {};
```

Comment:
wouldn't _contentState( ) guarantee to return an object? Is there other purpose to declare the object this way?